biogeochem/EDMortalityFunctionsMod.F90

```diff
@@ -83,8 +86,11 @@ subroutine mortality_rates( cohort_in,bc_in,cmort,hmort,bmort,frmort )
     
     ! Carbon Starvation induced mortality.
     if ( cohort_in%dbh  >  0._r8 ) then
-       call bleaf(cohort_in%dbh,cohort_in%pft,cohort_in%canopy_trim,b_leaf)
-       call storage_fraction_of_target(b_leaf, cohort_in%bstore, frac)
+
+       call bleaf(cohort_in%dbh,cohort_in%pft,cohort_in%canopy_trim,leaf_c_target)
```

Comment:
Or maybe I am accidentally conflating things since I also see this sub is called bleaf.

Reply:
that subroutine should be renamed to convey that it is returning the allometric target leaf carbon

Reply:
aha! that makes sense then.  I agree renaming would reduce confusion